### ISSUE TITLE
How long it takes to train mv_triplane_transformer_10K

### ISSUE THREAD

**Issue by throb081** [theEricMa/ScaleDreamer#6]
Hello，i want to run the Triplane-Transformer training code ,but i can't find out how much epoch it will take ,i have run for 3 days at A6000,it is still training .How long did it take when you run the training code ?
![屏幕截图 2024-08-23 153814](https://github.com/user-attachments/assets/67f799d3-2ca6-456f-9140-6ad0c3573c80)

**Comment by theEricMa** [comment]
Thanks for the question! The config specifies [100,000](https://github.com/theEricMa/ScaleDreamer/blob/3942bd20e5eb069ca803648b575c75c91e51fbdc/configs/multi-prompt_benchmark/asd_mv_triplane_transformer_10k.yaml#L122)  iterations, not epochs. However, due to gradient accumulation set to [2](https://github.com/theEricMa/ScaleDreamer/blob/3942bd20e5eb069ca803648b575c75c91e51fbdc/configs/multi-prompt_benchmark/asd_mv_triplane_transformer_10k.yaml#L129) , it effectively runs for 200,000 iterations. This explains the longer runtime you're observing.

**Comment by throb081** [comment]
Thanks！！！That's very kind of you！I‘m new on this field,there's a lot i don't know,.Your response helps me a lot!!
